packages/flutter_tools/bin/xcode_backend.dart

```diff
@@ -265,19 +265,19 @@ class Context {
           '-insert',
           'NSBonjourServices.0',
           '-string',
-          '_dartobservatory._tcp',
+          '_dartvmService._tcp',
```

Comment:
What reads this?

Reply:
This is for discovering the VM service URI via mDNS. The engine currently supports both `_dartobservatory._tcp` and `_dartvmService._tcp`.